README-tmp.md

```diff
@@ -0,0 +1,379 @@
+
+```bash
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_B}
+aws ec2 accept-vpc-peering-connection --vpc-peering-connection-id "${PEER_REQUEST_ID}"
+```
+
+3. Retrieve the route table IDs for Account A and add routes to Account B VPC:
+
+```bash
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_A}
+for NODE in $(oc get nodes --selector=node-role.kubernetes.io/worker | tail -n +2 | awk '{print $1}')
+do
+    SUBNET=$(aws ec2 describe-instances --filters "Name=private-dns-name,Values=$NODE" --query 'Reservations[*].Instances[*].NetworkInterfaces[*].SubnetId' | jq -r '.[0][0][0]')
+    echo SUBNET is ${SUBNET}
+    ROUTE_TABLE_ID=$(aws ec2 describe-route-tables --filters "Name=association.subnet-id,Values=${SUBNET}" --query 'RouteTables[*].RouteTableId' | jq -r '.[0]')
+    echo Route table ID is $ROUTE_TABLE_ID
+    aws ec2 create-route --route-table-id ${ROUTE_TABLE_ID} --destination-cidr-block ${AWS_ACCOUNT_B_VPC_CIDR} --vpc-peering-connection-id ${PEER_REQUEST_ID}
+done
+```
+
+4. Retrieve the route table IDs for Account B and add routes to Account A VPC:
+
+```bash
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_B}
+for ROUTE_TABLE_ID in $(aws ec2 describe-route-tables   --filters "Name=vpc-id,Values=${AWS_ACCOUNT_B_VPC_ID}"   --query "RouteTables[].RouteTableId" | jq -r '.[]')
+do
+    echo Route table ID is $ROUTE_TABLE_ID
+    aws ec2 create-route --route-table-id ${ROUTE_TABLE_ID} --destination-cidr-block ${AWS_ACCOUNT_A_VPC_CIDR} --vpc-peering-connection-id ${PEER_REQUEST_ID}
+done
+```
+
+5. Enable DNS resolution for Account A to access Account B's VPC:
+
+```bash
+aws ec2 modify-vpc-peering-connection-options --vpc-peering-connection-id ${PEER_REQUEST_ID} --accepter-peering-connection-options AllowDnsResolutionFromRemoteVpc=true
+```
+
+##  5.9.5.7 Configure security groups in Account B to allow NFS traffic from Account A to EFS
+
+1. Switch to your Account B profile:
+
+```bash
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_B}
+```
+
+2. Execute the following commands to configure the VPC security groups for EFS access:
+
+```bash
+SECURITY_GROUP_ID=$(aws ec2 describe-security-groups --filters Name=vpc-id,Values="${AWS_ACCOUNT_B_VPC_ID}" | jq -r '.SecurityGroups[].GroupId')
+aws ec2 authorize-security-group-ingress \
+ --group-id "${SECURITY_GROUP_ID}" \
+ --protocol tcp \
+ --port 2049 \
+ --cidr "${AWS_ACCOUNT_A_VPC_CIDR}" | jq .
+```
+
+##  5.9.5.8 Create a region-wide EFS filesystem in Account B
+
+1. Switch to your Account B profile:
+
+```bash
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_B}
+```
+
+2. Create a region-wide EFS File System:
+
+```bash
+CROSS_ACCOUNT_FS_ID=$(aws efs create-file-system --creation-token efs-token-1 \
+--region ${AWS_REGION} \
+--encrypted | jq -r '.FileSystemId') \
+&& echo $CROSS_ACCOUNT_FS_ID
+```
+
+3. Configure region-wide Mount Targets for EFS (this creates a mount point in each subnet of your VPC):
+
+```bash
+for SUBNET in $(aws ec2 describe-subnets \
+  --query 'Subnets[*].{SubnetId:SubnetId}' \
```

Comment:
```suggestion
  --filters "Name=vpc-id,Values=${AWS_ACCOUNT_B_VPC_ID}" \
  --query 'Subnets[*].{SubnetId:SubnetId}' \
```
We'd better filter with vpc id since current way will query all subnets in the region.